### PR TITLE
Move names of people to a new people file

### PR DESCRIPTION
**Pull request by jasongrout**
I think the governance structure should not hard-code people's names (other than perhaps Fernando).    This PR moves people and institutions' names to a new file.

### CODE REVIEW

**Comment by jasongrout** [comment]
I think it's going to be hard to keep track of Institutional Contributors, and I think that list will get stale especially easily.  What do people think about not listing the institutional contributor names, but just the institutions and institutional council members?

File: people.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# People in the governance structure
```
Comment (jasongrout) [inline people.md]: I don't really like this title.  Suggestions?

File: governance.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Main Governance Document
 
+The official version of this document, along with a list of
+individuals and institutions in the roles defined in the governance
+section below, is contained in The Project Governance Repository at:
+
+[https://github.com/jupyter/governance](https://github.com/jupyter/governance)
```
Comment (ellisonbg) [inline governance.md]: Hmm, isn't that a bit odd to have a document link to itself?

File: people.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+the governance.
+
+Benevolent Dictator for Life
+============================
+
+Fernando Perez is the Benevolent Dictator for Life (BDFL).
+
+Steering Council
+================
+
```
Comment (ellisonbg) [inline people.md]: For now I would alphabetize this list. Eventually, we may want to order by how long people have served.

File: people.md (hunk 1 above)
Comment (ellisonbg) [inline people.md]: Maybe Current "Steering Council and Institutional Partners"?

**Comment by ellisonbg** [comment]
Thanks Jason!

File: governance.md (hunk 2 above)
Comment (jasongrout) [inline governance.md]: It links to the governance repo, not the document.  I think it's important, especially when this document may be copied or displayed elsewhere, to say where the original is.

File: people.md (hunk 3 above)
Comment (jasongrout) [inline people.md]: Emacs says the list is in alphabetical order (by first name).  M-x sort-lines....

File: people.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- Kyle Kelley
+- Matthias Bussionnier
+- Min Ragan-Kelley
+- Thomas Kluyver
+
+NumFOCUS Subcommittee
+---------------------
+
+- Brian Granger
+- Fernando Perez
```
Comment (fperez) [inline people.md]: I'm not on the NumFocus one

File: people.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+- Jason Grout
+- Jessica Hamrick
+- Jonathan Frederic
+- Kyle Kelley
+- Matthias Bussionnier
+- Min Ragan-Kelley
+- Thomas Kluyver
+
+NumFOCUS Subcommittee
```
Comment (fperez) [inline people.md]: We call it the 'Fiscal oversight team'

File: people.md (hunk 3 above)
Comment (fperez) [inline people.md]: +1 to alphabetical as per `M-x sort-lines` for simplicity, we could just add in parens a year of joining, which could contain a year of leaving for folks who leave.

File: people.md (hunk 5 above)
Comment (jasongrout) [inline people.md]: It's not called that in the governance document.  How about we make a separate pull request to change the name of the committee in both documents?

Comment (fperez) [inline people.md]: We had originally called it the fiscal oversight team, to indicate that was its narrow remit. If people want to rename it to "NF subcommitte", I don't really care. But "NF subcommittee" is not historically accurate, at least, that's all :)

Before bothering with a separate PR, I'm happy to hear if people prefer a rename. I don't particularly care, I was just surprised to see this name, since that wasn't what we called it.

File: people.md (hunk 4 above)
Comment (fperez) [inline people.md]: BTW, who are the current signatories of the NF one at this point? We had to re-sign when we ended up with too many folks at Berkeley effectively working under me... I forgot the list...

File: people.md (hunk 5 above)
Comment (ellisonbg) [inline people.md]: I am fine with the original name. Based on discussion that are happening in NumFOCUS the structure of these teams and relationship to project leadership and governance may change over time - but we aren't there yet.

Comment (rgbkrk) [inline people.md]: I believe the Fiscal oversight team has changed as well, now that most of the group is at Berkeley. From an email with @takluyver, he asked if Jon and I would like to be on the fiscal oversight team, replacing @Carreau and @fperez.

File: people.md (hunk 4 above)
Comment (ellisonbg) [inline people.md]: I think it is Jon, Kyle, Myself, Thomas and Stefan, but I am not positive.

File: people.md (hunk 5 above)
Comment (jasongrout) [inline people.md]: Okay, how about we use this list of names for now, which is hardcoded in the governance document right now.  Then we open a new PR for updating the list of names.

This PR is about "splitting the repo".  Then we can update the "name repo" in a separate PR.

**Comment by jasongrout** [comment]
I propose that we accept this PR as-is, as strictly a "split out the currently-hardcoded names" PR (plus some grammar and typo cleanup).  Let's open a new PR for updating names.

**Comment by jasongrout** [comment]
Now the question is: do we have a vote on each PR modifying the governance documents?  That would be safest, I think.  Maybe after a PR is reviewed, the final commit of the PR should be proposed as an official vote to the steering council.

**Comment by rgbkrk** [comment]
:+1: to this PR. Let's merge it.

**Comment by ellisonbg** [comment]
In general, we haven't used formal votes with the project in the past. I
think informal consensus is a much better model as it encourages us to find
solutions that more people are happy with. Votes can lead to a culture
where 49.99999% of the people are unhappy about each decision.

But I do think we should discuss these questions in a separate PR on this
repo :)

But until we do that I think informal consensus of the Steering Council is
fine.



On Fri, Sep 4, 2015 at 11:37 AM, Jason Grout <notifications@github.com>
wrote:

> Now the question is: do we have a vote on each PR modifying the governance
> documents? That would be safest, I think. Maybe after a PR is reviewed, the
> final commit of the PR should be proposed as an official vote to the
> steering council.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/governance/pull/5#issuecomment-137814822>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by ellisonbg** [comment]
Oh and I am fine with the merging the current state of this PR.

On Fri, Sep 4, 2015 at 11:49 AM, Brian Granger <ellisonbg@gmail.com> wrote:

> In general, we haven't used formal votes with the project in the past. I
> think informal consensus is a much better model as it encourages us to find
> solutions that more people are happy with. Votes can lead to a culture
> where 49.99999% of the people are unhappy about each decision.
>
> But I do think we should discuss these questions in a separate PR on this
> repo :)
>
> But until we do that I think informal consensus of the Steering Council is
> fine.
>
>
>
> On Fri, Sep 4, 2015 at 11:37 AM, Jason Grout <notifications@github.com>
> wrote:
>
>> Now the question is: do we have a vote on each PR modifying the
>> governance documents? That would be safest, I think. Maybe after a PR is
>> reviewed, the final commit of the PR should be proposed as an official vote
>> to the steering council.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/jupyter/governance/pull/5#issuecomment-137814822>.
>>
>
>
>
> --
> Brian E. Granger
> Associate Professor of Physics and Data Science
> Cal Poly State University, San Luis Obispo
> @ellisonbg on Twitter and GitHub
> bgranger@calpoly.edu and ellisonbg@gmail.com
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by jasongrout** [comment]
As might be obvious, +1 for merging this PR.

**Comment by Carreau** [comment]
OK, merging and iterating.

**Comment by damianavila** [comment]
For the record :+1: as well to merge it.
